File: src/test/java/se/johannesdahlgren/aoc25/Day5Test.java
package se.johannesdahlgren.aoc25;

import org.junit.jupiter.api.Test;

import static org.assertj.core.api.Assertions.assertThat;

class Day5Test {
  @Test
  void part1Example() {
    Day5 day5 = new Day5("example");
    assertThat(day5.numberOfFreshIngredients()).isEqualTo(3);
  }

  @Test
  void part1() {
    Day5 day5 = new Day5("");
    assertThat(day5.numberOfFreshIngredients()).isEqualTo(868);
  }
}

File: src/test/kotlin/se/johannesdahlgren/aoc22/Day5Test.kt
package se.johannesdahlgren.aoc22

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class Day5Test {
    @Test
    fun part1Example() {
        val day5 = Day5("day5Example")
        assertThat(day5.getCratesOnTop()).isEqualTo("CMZ")
    }

    @Test
    fun part1() {
        val day5 = Day5("day5")
        assertThat(day5.getCratesOnTop()).isEqualTo("FZCMJCRHZ")
    }

    @Test
    fun part2Example() {
        val day5 = Day5("day5Example")
        assertThat(day5.getCratesOnTopMultiCrate()).isEqualTo("MCD")
    }

    @Test
    fun part2() {
        val day5 = Day5("day5")
        assertThat(day5.getCratesOnTopMultiCrate()).isEqualTo("JSDHQMZGF")
    }

}